From 7006e5fade520e79ee795ff7a205452dbaec0f19 Mon Sep 17 00:00:00 2001
From: Thibault Saunier <thibault.saunier@collabora.com>
Date: Thu, 24 Jan 2013 16:23:32 -0300
Subject: [PATCH 7/9] Make it compile on IOS

Simply comment AC_FUNC_MALLOC in configure.ac. Explanation here:
http://nerdland.net/unstumping-the-internet/malloc-has-not-been-declared/
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index d4caa5d..5053626 100644
--- a/configure.ac
+++ b/configure.ac
@@ -196,7 +196,7 @@ AM_CONDITIONAL([HAVE_SSE], [test "x$have_sse_intrinsics" = "xyes"])
 dnl ############################################################################
 dnl # Checks for library functions/classes                                     #
 dnl ############################################################################
-AC_FUNC_MALLOC
+# AC_FUNC_MALLOC
 AC_TYPE_SIGNAL
 
 dnl make -lm get added to the LIBS
-- 
2.1.4

